tests/hold-release/05-release.t

```diff
@@ -63,7 +63,7 @@ cylc release ${CYLC_SUITE_NAME} 'holdrelease.1'
     [[stop]]
         inherit = STOP
         script = """
-        sleep 5  # make sure other tasks are finished and gone
+        cylc__job__poll_grep_suite_log 'Removing succeeded task dog1.1'
```

Comment:
Is this the improvement you mentioned some other day that we could do in our functional tests? Using this function instead of sleeping? If so I will try to keep this in mind whenever I touch a functional test, checking if it uses sleep, and trying to replace if possible.

Reply:
Yes it's one of the things to do, to avoid relying on hope-for-the-best sleep intervals that may be wrong when the host is under extreme load.